NeuroID/TrackerClassExtensions/UIClassExtensions/UIButton.swift

```diff
@@ -49,7 +49,7 @@ import UIKit
 //            let clickTG = ParamsCreator.getTGParamsForInput(eventName: NIDEventName.click, view: self, type: NIDEventName.click.rawValue, attrParams: ["v": lengthValue, "\(Constants.hashKey.rawValue)": self.titleLabel?.text])
 //            var clickEvent = NIDEvent(type: NIDEventName.click, tg: clickTG)
 //
-//            let screenName = self.className ?? UUID().uuidString
+//            let screenName = self.className ?? ParamsCreator.genId()
```

Comment:
Looks like this file is completely commented, can we delete it?

Reply:
That is true,  we have left this file to see if we want to work on the UIButton in the future